src/backend/common/MemoryManager.hpp

```diff
@@ -14,6 +14,8 @@
 #include <util.hpp>
 
 #include <algorithm>
+
+// TODO(umar): Remove iostream
 #include <iomanip>
 #include <iostream>
```

Comment:
isn't that only a cause of concern if we are doing `using namespace std` inside this header(which we are not right now) ?